### ISSUE TITLE
Resources$NotFoundException when inflating WebView on Android 5.x

### ISSUE THREAD

**Issue by b95505017** [intercom/intercom-android#456]
It seems like the same issue like #332 which inflate WebView through XML on Android 5.x, causing crash when entering help center.

```
   at io.intercom.android.sdk.activities.IntercomHelpCenterActivity.onCreate(IntercomHelpCenterActivity.java:86)
   at android.app.Activity.performCreate(Activity.java:5937)
   at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1105)
   at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2251)
   at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2360) 
   at android.app.ActivityThread.access$800(ActivityThread.java:144) 
   at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1278) 
   at android.os.Handler.dispatchMessage(Handler.java:102) 
   at android.os.Looper.loop(Looper.java:135) 
   at android.app.ActivityThread.main(ActivityThread.java:5221) 
   at java.lang.reflect.Method.invoke(Native Method) 
   at java.lang.reflect.Method.invoke(Method.java:372) 
   at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:899) 
   at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:694) 
Caused by: java.lang.reflect.InvocationTargetException
   at java.lang.reflect.Constructor.newInstance(Native Method)
   at java.lang.reflect.Constructor.newInstance(Constructor.java:288)
   at android.view.LayoutInflater.createView(LayoutInflater.java:607)
   at com.android.internal.policy.impl.PhoneLayoutInflater.onCreateView(PhoneLayoutInflater.java:55) 
   at android.view.LayoutInflater.onCreateView(LayoutInflater.java:682) 
   at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:741) 
   at android.view.LayoutInflater.rInflate(LayoutInflater.java:806) 
   at android.view.LayoutInflater.rInflate(LayoutInflater.java:809) 
   at android.view.LayoutInflater.inflate(LayoutInflater.java:504) 
   at android.view.LayoutInflater.inflate(LayoutInflater.java:414) 
   at android.view.LayoutInflater.inflate(LayoutInflater.java:365) 
   at android.support.v7.app.AppCompatDelegateImplV9.setContentView(AppCompatDelegateImplV9.java:287) 
   at android.support.v7.app.AppCompatActivity.setContentView(AppCompatActivity.java:139) 
   at io.intercom.android.sdk.activities.IntercomHelpCenterActivity.onCreate(IntercomHelpCenterActivity.java:86) 
   at android.app.Activity.performCreate(Activity.java:5937) 
   at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1105) 
   at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2251) 
   at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2360) 
   at android.app.ActivityThread.access$800(ActivityThread.java:144) 
   at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1278) 
   at android.os.Handler.dispatchMessage(Handler.java:102) 
   at android.os.Looper.loop(Looper.java:135) 
   at android.app.ActivityThread.main(ActivityThread.java:5221) 
   at java.lang.reflect.Method.invoke(Native Method) 
   at java.lang.reflect.Method.invoke(Method.java:372) 
   at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:899) 
   at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:694) 
Caused by: android.content.res.Resources$NotFoundException: String resource ID #0x2040002
   at android.content.res.Resources.getText(Resources.java:274)
   at android.content.res.Resources.getString(Resources.java:360)
   at com.android.org.chromium.content.browser.ContentViewCore.setContainerView(ContentViewCore.java:702)
   at com.android.org.chromium.content.browser.ContentViewCore.initialize(ContentViewCore.java:608)
   at com.android.org.chromium.android_webview.AwContents.createAndInitializeContentViewCore(AwContents.java:619)
   at com.android.org.chromium.android_webview.AwContents.setNewAwContents(AwContents.java:758)
   at com.android.org.chromium.android_webview.AwContents.<init>(AwContents.java:608)
   at com.android.org.chromium.android_webview.AwContents.<init>(AwContents.java:546)
   at com.android.webview.chromium.WebViewChromium.initForReal(WebViewChromium.java:312)
   at com.android.webview.chromium.WebViewChromium.access$100(WebViewChromium.java:97)
   at com.android.webview.chromium.WebViewChromium$1.run(WebViewChromium.java:264)
   at com.android.webview.chromium.WebViewChromium$WebViewChromiumRunQueue.drainQueue(WebViewChromium.java:124)
   at com.android.webview.chromium.WebViewChromium$WebViewChromiumRunQueue$1.run(WebViewChromium.java:111)
   at com.android.org.chromium.base.ThreadUtils.runOnUiThread(ThreadUtils.java:144)
   at com.android.webview.chromium.WebViewChromium$WebViewChromiumRunQueue.addTask(WebViewChromium.java:108)
	at com.android.webview
```

**Comment by DaleCantwell** [comment]
Hi @b95505017 
Did you encounter this on a simulator or a real device? 
Did this device have play services?
Can you tell if this was on a custom rom?

Digging into this issue I can reliably reproduce it on a simulator where I have no webview thus would not be able to inflate one.

**Comment by b95505017** [comment]
I meet this issue on (Android 5.0 Google APIs) official emulator.

**Comment by ronocod** [comment]
Fixed in 4.1.8.
